pkg/cpi/mockingUtils.go

```diff
@@ -28,6 +28,24 @@ func GetCPIFunctionMockResponse(functionName, testType string) (*http.Response,
 					}`))),
 		}
 		return &res, errors.New("Internal Server Error")
+	case "IntegrationArtifactUpdateConfiguration":
+		if testType == "Positive" {
+			return GetEmptyHTTPResponseBody()
+		}
+		if testType == "Negative_With_ResponseBody" {
+			return GetNegativeCaseHTTPResponseBody()
```

Comment:
I would like to suggest changing this return to `return GetNegativeCaseHTTPResponseBody(), nil` and changing the function return values accordingly. Otherwise we should change the name of the function to something like: `GetNegativeCaseHTTPResponseBodyAndErrorNil`